inst/templates/basic-project.R.template

```diff
@@ -1,9 +1,10 @@
 
 ## environment setup
-setPaths(cachePath = "cache",
-         inputPath = "inputs",
-         modulePath = "modules",
-         outputPath = "outputs")
+projectPath <- "{{{projPath}}}"
```

Comment:
we should only be using relative paths here, otherwise this becomes non-portable. I.e., remove `projectPath`

Reply:
done